### ISSUE TITLE
Source of Gender, Age, and Race training data?

### ISSUE THREAD

**Issue by liechtym** [serengil/deepface#628]
Hello! Thanks for the great repository.

I have a quick question. What datasets did you use to train for age, gender, and race recognition? I noticed you said it was based on the VGG-Face model, but I'm not where the specific gender, age, and race labels came from. My purpose in asking this question is to make sure that my validation set doesn't interlap with the training set of this tool.

Thanks,
Matthew

**Comment by serengil** [comment]
If you follow the read-me of the repo, you will be redirected to its tutorials. In those tutorials, reference datasets were mentioned already.
